### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
**This update:**
- Improves interactivity
- Adds a new label with links to go
- Improves the external presentation of information
- Adds a description and some facts.

### Related issue(s)/PR(s)
NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,87 @@
-# MODX Revolution
-
-[![Build Status](https://travis-ci.org/modxcms/revolution.svg?branch=2.x)](https://travis-ci.org/modxcms/revolution) [![Slack Chat](https://img.shields.io/badge/chat_in_slack-online-green.svg?longCache=true&style=flat&logo=slack)](https://modx.org)
+<p align="center">
+  <a href="http://modx.com/">
```
Comment (JoshuaLuckers) [inline README.md]:
```suggestion
  <a href="https://modx.com/">
```

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jako) [reply README.md]: https://

Comment (Mark-H) [reply README.md]: Missing a `:` there @JoshuaLuckers

Comment (Ruslan-Aleev) [reply README.md]: @Mark-H @JoshuaLuckers Maybe it makes sense for all links to write without a certificate?
For example:
`<a href="//modx.com/">`

**Comment by JoshuaLuckers** [comment]
Personally I'm not a big fan of some of the additions. 
- To much HTML in a markdown document. 
- To many new badges and external images.
- Are the emoji really needed?

**Comment by JoshuaLuckers** [comment]
And the list of "statistics" should be fact checked if others agree with keeping them.

**Review by JoshuaLuckers — APPROVED** [review]

**Comment by Ibochkarev** [comment]
@JoshuaLuckers The list for statistics is taken from here:
- https://modx.com/
- https://modx.com/extras/

If there are corrections, let's fix it.

Yes, and I'm for emoji =)

**Review by JoshuaLuckers — COMMENTED** [review]
Accidentally approved it, it needs more discussion and input from other reviewers.

**Comment by Jako** [comment]
> Accidentally approved it.

Just request changes to disapprove it.

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]
This PR needs approval of other integrators (i cant undo the approval).

**Comment by JoshuaLuckers** [comment]
Thanks for taking time to create this PR. We are satisfied with the current read me file, therefore I’m closing this PR.

**Comment by OptimusCrime** [comment]
Closed already? I really liked the header and the more narrow width of the text. I would like to see that included. I do not like the emojis, but some of the other things could be a nice addition. Right now, the README does not sell the product as well as it could here on GitHub.

**Comment by Ibochkarev** [comment]
@JoshuaLuckers I can make the necessary changes and not use HTML

**Comment by JoshuaLuckers** [comment]
Maybe you can open an issue first suggesting the changes @Ibochkarev 😄

**Comment by Ibochkarev** [comment]
@JoshuaLuckers I can't because this PR is closed by you.

**Comment by OptimusCrime** [comment]
@Ibochkarev Open a new issue and a new PR.

**Comment by Ibochkarev** [comment]
@JoshuaLuckers @OptimusCrime #14476

**Pull request closed without merging**